js/form.js

```diff
@@ -1,7 +1,23 @@
 const contactForm = document.querySelector('.form-action');
 const error = document.querySelector('small');
+const formInput = document.querySelectorAll('.form-data');
 const email = document.querySelector('#email');
 
+let form = {
+  fullName: formInput[0].value,
+  email: formInput[1].value,
+  comment: formInput[2].value,
+};
+
+formInput.forEach((input) => {
+  input.addEventListener('change', () => {
+    form.fullName = formInput[0].value;
+    form.email = formInput[1].value;
+    form.comment = formInput[2].value;
+    localStorage.setItem('form', JSON.stringify(form));
+  });
+});
```

Comment:
No data is stored when some data is fed to the form fields, and data does not remain in browser after the page is refreshed. Just like mentioned in the previous review, kindly ensure that:
- [ ] Data entered into the form fields are persisted to the local storage on the fly.
- [ ] Persisted data should reflect in the form fields, even when the page is refreshed.

**Entered data not reflecting in local storage:**

![image](https://user-images.githubusercontent.com/12745474/209294648-2b2ccbda-a2b2-4bd0-93fd-d57bcaa27188.png)

![image](https://user-images.githubusercontent.com/12745474/209295376-07450087-6ec2-4bbd-a327-9fdea59ac018.png)